snapm/manager/plugins/lvm2.py

```
@@ -195,6 +195,41 @@
+
+
+def _dm_major() -> int:
+    """
+    Teturn the device-mapper major device number for the running
+    system.
+
+    :returns: The device-mapper major number as an int.
+    :raises: ``SnapmNotFoundError`` if device-mapper is not found
+             in ``/proc/devices``.
+    """
+    if _dm_major > 0:
```

Ruff (0.12.2)

Local variable `_dm_major` referenced before assignment

(F823)